src/main/java/frc/robot/vision/data/VisionData.java

```diff
@@ -0,0 +1,23 @@
+package frc.robot.vision.data;
+
+import edu.wpi.first.math.geometry.Pose3d;
+
+public class VisionData {
```

Comment:
This is not that critical, but this can be a record

Reply:
you cannot inherite from a record

Reply:
The ApriltagVisionData needs to inherit from the VisionData and it is not possible to inherit from a record

Reply:
@tnrz135 can I resolve?